src/TileManager.js

```diff
@@ -120,20 +120,29 @@ export function getTileUrl(urlTemplate, data) {
  * @param {object} layer leaflet tilelayer
  * @param {object} bounds L.bounds
  * @param {number} zoom zoomlevel 0-19
+ * @param {L.CRS} [crs] to calculate the world pixel bounds for TMS scheme
  *
  * @return {Array.<tileInfo>}
  */
-export function getTileUrls(layer, bounds, zoom) {
+export function getTileUrls(layer, bounds, zoom, crs = L.CRS.EPSG3857) {
```

Comment:
https://github.com/allartk/leaflet.offline/blob/master/test/TileManagerTest.js#L3 This function is a critical part, I would like to see an additional test for it.